content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -245,14 +238,14 @@ taint is removed before that time, the pod will not be evicted.
 
 <!--
 ## Example Use Cases
--->
-## 使用例子
 
-<!--
 Taints and tolerations are a flexible way to steer pods *away* from nodes or evict
 pods that shouldn't be running. A few of the use cases are
 -->
-通过污点和容忍度，可以灵活地让 Pod **避开** 某些节点或者将 Pod 从某些节点驱逐。下面是几个使用例子：
+## 使用例子
+
+通过污点和容忍度，可以灵活地让 Pod **避开** 某些节点或者将 Pod 从某些节点驱逐。
```

Comment:
```suggestion
通过污点和容忍度，可以灵活地让 Pod **避开**某些节点或者将 Pod 从某些节点驱逐。
```

Reply:
Not sure if the stars will work if we remove the space.